### PR TITLE
extend unit test for insertClauseShardingConditionEngine(#16102)

### LINKED ISSUE #16102: Add unit test for InsertClauseShardingConditionEngine#createShardingConditions
Hi community,

This issue is for [OpenForce](https://github.com/apache/shardingsphere/discussions/12683#discussioncomment-2139441)

### PR DESCRIPTION
**Pull request by pparthenhs**
Fixes #16102

### CODE REVIEW

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/route/engine/condition/engine/InsertClauseShardingConditionEngineTest.java [hunk 1]
```diff
@@ -34,52 +39,90 @@
...
-    private final InsertClauseShardingConditionEngine insertClauseShardingConditionEngine = new InsertClauseShardingConditionEngine(mock(ShardingRule.class), mock(ShardingSphereSchema.class));
-    
+
+    private InsertClauseShardingConditionEngine insertClauseShardingConditionEngine;
+
+    private ShardingRule shardingRule;
+
     @Mock
     private InsertStatementContext insertStatementContext;
-
```
Comment (strongduanmu) [inline shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/route/engine/condition/engine/InsertClauseShardingConditionEngineTest.java:56]: @pparthenhs Please keep same indents with previous line.

**Review by strongduanmu — CHANGES_REQUESTED** [review]
Hi @pparthenhs, thank you for your contribution, can you take a look at ci error?

**Comment by pparthenhs** [comment]
Hello @strongduanmu, I fixed the CI error. So we could proceed with the PR process and merged.

**Review by strongduanmu — CHANGES_REQUESTED** [review]
@pparthenhs Please solve review suggestion.

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/route/engine/condition/engine/InsertClauseShardingConditionEngineTest.java (hunk 1 above)
Comment (pparthenhs) [reply shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/route/engine/condition/engine/InsertClauseShardingConditionEngineTest.java:56]: This is a file format for the related issue, regarding CRLF and LF. In addition, this code line moved as the code added above.

Please me know how to proceed. 

Really appreciate your time, support and collaboration.

**Comment by pparthenhs** [comment]
> @pparthenhs Please solve review suggestion.

This is a file format for the related issue, regarding CRLF and LF. In addition, this code line moved as the code added above.

Please me know how to proceed.

Really appreciate your time, support, and collaboration.

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**